python/lsst/faro/measurement/DetectorTableMeasurement.py

```diff
@@ -0,0 +1,129 @@
+import os
+
+import lsst.pipe.base as pipeBase
+from lsst.verify.tasks import MetricConnections
+import lsst.pex.config as pexConfig
+import lsst.geom
+from lsst.utils import getPackageDir
+
+from lsst.faro.base.CatalogMeasurementBase import CatalogMeasurementBaseTaskConfig, CatalogMeasurementBaseTask
+from lsst.pipe.tasks.loadReferenceCatalog import LoadReferenceCatalogConfig, LoadReferenceCatalogTask
+
+__all__ = ("DetectorTableMeasurementTaskConfig", "DetectorTableMeasurementTask")
+
+
+class DetectorTableMeasurementTaskConnections(MetricConnections,
+                                              dimensions=("instrument", "visit", "detector", "band"),
+                                              defaultTemplates={"refDataset": ""}):
+
+    catalog = pipeBase.connectionTypes.Input(doc="Source catalog for visit.",
+                                             dimensions=("instrument", "visit", "band"),
+                                             storageClass="DataFrame",
+                                             name="sourceTable_visit",
+                                             deferLoad=True)
+
+    refcat = pipeBase.connectionTypes.PrerequisiteInput(
+        doc="Reference catalog.",
+        name="{refDataset}",
+        storageClass="SimpleCatalog",
+        dimensions=("skypix",),
+        deferLoad=True,
+        multiple=True
+    )
+
+    measurement = pipeBase.connectionTypes.Output(doc="Per-detector measurement.",
```

Comment:
If John prefers the other format, we can go that way.  I just want consistency.  For the record we use this format in the other files, I thnk

Reply:
I do prefer the format with newline and all single-indent. Agree with consistency. We will probably want to update all the class names for connections and configs to follow DM convention, so that might be a time to make this all consistent throughout the package.